CIP-1694/README.md

```diff
@@ -0,0 +1,624 @@
+---
+CIP: 1694
+Title: An On-Chain Decentralized Governance Mechanism for Voltaire
+Status: Proposed
+Category: Ledger
+Authors:
+    - Jared Corduan <jared.corduan@iohk.io>
+    - Matthias Benkort <matthias.benkort@cardanofoundation.org>
+    - Kevin Hammond <kevin.hammond@iohk.io>
+    - Charles Hoskinson <charles.hoskinson@iohk.io>
+    - Samuel Leathers<samuel.leathers@iohk.io>
+Discussions:
+    - https://github.com/cardano-foundation/cips/pulls/?
+Created: 2022-11-18
+License: CC-BY-4.0
+---
+
+# An On-Chain Decentralized Governance Mechanism for Voltaire
+
+## Abstract
+
+We propose a revision of Cardano's on-chain governance system to support the new requirements for Voltaire. The existing specialized governance support for protocol parameter updates and MIR certificates will be deprecated, and two new fields will be added to normal transaction bodies:
+
+1. governance actions;
+2. votes.
+
+**Any Cardano user** will be allowed to submit a **governance action**. Three distinct groups will be responsible for ratifying these governance actions using their **votes**:
+
+1. a Constitutional committee;
+2. a group of delegation representatives (henceforth called **DReps**); and
+3. the stake pool operators (henceforth called **SPOs**).
+
+Ratified actions may then be **enacted** on-chain, following a set of well-defined rules.
+
+As with stake pools, any Ada holder may register to be a DRep and so
+choose to represent themselves if they wish, or they may, instead, delegate their voting rights
+to any other registered representative.  These voting rights will be based on Ada holdings.
```

Comment:
I think points 5 and 6 are outside the scope of this CIP, the US constitution doesn't lay out Congress customs and proceedings in it, just what they are responsible for. 

Just an idea about points 1-3. Maybe the framework laid out in this CIP for the constitutional committee should be generalized to create a committee for each governance action e.g. Treasury Committee, Parameter Update Committee(this should probably be broken up even further), Hard Fork Committee. And the CC can keep it's veto role over all other committees, with this veto power I feel like it functions more like the executive branch.

Reply:
> what is even the point with having representatives?

> idea with representation is that the person who represent you have more time and expertise to make better decisions on your behalf.

We want everyone to have the option to vote for themselves, but exactly as you say, there are good reasons to delegate this right.

> the system incentives active participation

Indeed this is extraordinarily important, but not rewarding participation comes with its own perverse incentives. This part of the design is still undecided, and is being thought about by the research team. It probably merits it's own CIP.

Reply:
> This text doesn't describe what kind of forum (if any) there will be for the decisionmakers to get together, vet ideas, make their cases, argue points, etc. Maybe this has been left out purposly, either because such a forum isn't wanted or because it's too early at this stage, but if there are plans to have such a structure, how will the system decide which actors are included, if anyone holding ada is/could be a representative?

Indeed it was left out because it is too early. It will be something off chain, something perhaps the community itself could build. I would hope that everyone agrees that _everyone_ should be included who wants to be.

Reply:
Yeah, for sure, my wording is a bit weird, but what I'm getting at is that leaving it undefined opens up for many conversations that needs to take place in the open to happen privately, since that is probably what happens atm. There is ofc. nothing stopping this from happening no mather what system you implement.

There could be structures, such as the CIP processes, where a CIP _needs_ to be introduced to the open before being acted on. Maybe this is in the plans? Or maybe to rigid of a system for certain categories of decisions.

Reply:
We are definitely very aware that a "voting center" of some kind is a necessity. A CIP or a CRC is probably a great idea, and one may even be in the works already.